src/command.rs

```diff
@@ -61,13 +61,14 @@ impl CmdThread {
   }
 
   fn send(&self, msg: Msg) {
-    self.msg_tx.send(msg).ok().expect("command thread died");
+    self.msg_tx.unbounded_send(msg).ok().expect("command thread died");
   }
 }
 
 impl Drop for CmdThread {
   fn drop(&mut self) {
-    self.kill_tx.take().expect("CmdThread already killed.").complete(());
+    self.kill_tx.take().expect("CmdThread already killed.").send(()).expect(
+      "Input thread died prematurely.");
```

Comment:
This would be good - except I see I messed up earlier. Could you change "Input" to "Command" on this one and the one below while you're at it please?